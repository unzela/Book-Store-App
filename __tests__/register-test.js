import 'react-native';
import React from 'react';
import Register from '../src/register';
import renderer from 'react-test-renderer';

test('Register Snapshot',()=>{
    const snap = renderer.create(
        <Register/>
    ).toJSON();

    expect(snap).toMatchSnapshot();
});

test('Register Renderer',()=>{
    renderer.create(
        <Register/>
    );

  
});
