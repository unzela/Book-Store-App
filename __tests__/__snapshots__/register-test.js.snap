// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "lightgrey",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fb5b5a",
        "fontSize": 50,
        "fontWeight": "bold",
        "marginBottom": 40,
      }
    }
  >
    Sign Up
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 20,
        "padding": 20,
        "width": "80%",
      }
    }
  >
    <TextInput
      placeholder="Name..."
      placeholderTextColor="#003f5c"
      style={
        Object {
          "color": "black",
          "height": 50,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 20,
        "padding": 20,
        "width": "80%",
      }
    }
  >
    <TextInput
      placeholder="Email..."
      placeholderTextColor="#003f5c"
      style={
        Object {
          "color": "black",
          "height": 50,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 20,
        "padding": 20,
        "width": "80%",
      }
    }
  >
    <TextInput
      placeholder="Password..."
      placeholderTextColor="#003f5c"
      secureTextEntry={true}
      style={
        Object {
          "color": "black",
          "height": 50,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 20,
        "padding": 20,
        "width": "80%",
      }
    }
  >
    <TextInput
      placeholder="Mobile Number...."
      placeholderTextColor="#003f5c"
      style={
        Object {
          "color": "black",
          "height": 50,
        }
      }
    />
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fb5b5a",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginBottom": 10,
        "marginTop": 40,
        "opacity": 1,
        "width": "80%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
        }
      }
    >
      Create Account
    </Text>
  </View>
</View>
`;
